### PR TITLE
docs: cleanup all whitenoise

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

/kind cleanup

### CODE REVIEW

**Review by bsdnet — APPROVED** [review]
Thanks for cleaning them up.

**Review by neolit123 — COMMENTED** [review]
/ok-to-test
/kind cleanup
/priority backlog

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Review by dixudx — APPROVED** [review]

**Review by kensipe — COMMENTED** [review]
/lgtm

**Comment by bsdnet** [comment]
/lgtm

thanks for the clean up.

I am wonder whether we can write some verify test to make sure new commits do not create such issue in future.

**Comment by daixiang0** [comment]
ping @neolit123

**Comment by neolit123** [comment]
@daixiang0 i cannot merge / approve this PR.
see the bot comment above.

**Comment by daixiang0** [comment]
ping @fedebongio

**Comment by daixiang0** [comment]
ping @neolit123

**Comment by neolit123** [comment]
Try asking in the #pr-reviews channel of k8s slack.

File: build/go-runner/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Kubernetes go-runner image
```
Comment (justaugustus) [inline build/go-runner/README.md:1]: Remove this file.
The go-runner image building is now described here: https://github.com/kubernetes/release/blob/master/images/build/go-runner/README.md

**Review by justaugustus — CHANGES_REQUESTED** [review]
@daixiang0 -- thanks for the docs cleanup!

Can you rebase this? That should take care of the one review comment I have.
Everything else looks good to me once that is fixed.

**Comment by puerco** [comment]
Pinged the author on slack to see if we're moving forward with this one.

**Comment by daixiang0** [comment]
ping @puerco

**Comment by puerco** [comment]
Thank you @daixiang0 ! it is now correctly rebased. As your PR edits documents across a very wide spectrum of the project, it has to get approval from a lot of people in order to merge (check out the message from the bot). As neolit said above, the next step will be to request approvals from #pr-reviews.

/lgtm from my part

**Review by hasheddan — APPROVED** [review]
/lgtm

**Pull request closed without merging**